Detectors/PHOS/calib/PHOSCalibWorkflow/src/PHOSRunbyrunCalibDevice.cxx

```diff
@@ -0,0 +1,379 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "PHOSCalibWorkflow/PHOSCalibCollector.h"
+#include "DataFormatsPHOS/TriggerRecord.h"
+#include "Framework/ConfigParamRegistry.h"
+#include "DetectorsCalibration/Utils.h"
+#include "Framework/ControlService.h"
+
+#include "FairLogger.h"
+#include <fstream> // std::ifstream
+
+using namespace o2::phos;
+
+void PHOSCalibCollector::init(o2::framework::InitContext& ic)
```

Comment:
is this is a stray file? It is not used and the contents looks like a copy-paste from PHOSCalibCollector.cxx

Reply:
This is a seed for another one calibration process - Run-by-run calibration. Lets keep it as is till next PR.

Reply:
@peressounko But it is an exact clone of already existing PHOSCalibCollector.cxx. Why do you need to commit a dead cloned code? Moreover, that the PHOSCalibCollector itself should be splitted to calib digits extractor (though it could be probably done and optional output of clustering device) and aggregator parts (time-slots based).
So, what is the expected calibration info output size per TF?